config/defaults.mk

```diff
@@ -216,9 +216,10 @@ SUITESPARSE_LIB = -Wl,-rpath,$(SUITESPARSE_DIR)/lib -L$(SUITESPARSE_DIR)/lib\
  -lsuitesparseconfig $(LIB_RT) $(METIS_LIB) $(LAPACK_LIB)
 
 # SuperLU library configuration
-SUPERLU_DIR = @MFEM_DIR@/../SuperLU_DIST_5.1.0
-SUPERLU_OPT = -I$(SUPERLU_DIR)/SRC
-SUPERLU_LIB = -Wl,-rpath,$(SUPERLU_DIR)/lib -L$(SUPERLU_DIR)/lib -lsuperlu_dist_5.1.0
+SUPERLU_DIR = @MFEM_DIR@/../SuperLU_DIST_6.3.1
+SUPERLU_OPT = -I$(SUPERLU_DIR)/include
+SUPERLU_LIB = -Wl,-rpath,$(SUPERLU_DIR)/lib64 -L$(SUPERLU_DIR)/lib64 -lsuperlu_dist -lblas
```

Comment:
superlu_dist also requires lapack since version 6, for explicit inversion of the diagonal blocks, unless explicitly disabled

Reply:
Yes, I disabled it in my build in order to not add the dependence to MFEM.